src/Agent/NewRelic/Agent/Core/AgentHealth/AgentHealthReporter.cs

```diff
@@ -63,9 +63,9 @@ public override void Dispose()
 
         private void LogRecurringLogs()
         {
-            foreach (var data in _recurringLogDatas)
+            foreach (var logMessage in _recurringLogData)
             {
-                data?.LogAction(data.Message);
+                Log.Info(logMessage);
```

Comment:
It seems like this log data was previously logged at the Debug level.

Reply:
This also seems like an agent debugging feature that was introduced to make certain types of log messages repeat with every single harvest. I'm not sure that this is a feature that we need to continue to support.

Reply:
Yeah I can see how "Hey this wrapper is busted" is something you might want to report periodically so it doesn't get missed, but hopefully that's not something that happens often any more. I'd also be ok with removing it.

Reply:
I'm fine with either removing it, or updating the log level.